README.md

```diff
@@ -1,12 +1,52 @@
 [![Bolt.new: AI-Powered Full-Stack Web Development in the Browser](./public/social_preview_index.jpg)](https://bolt.new)
 
+# Bolt.new Fork by Cole Medin
```

Comment:
Based on this README this seems to be an attempt to merge changes made by Cole's fork into the original Bolt.new project. This is a ton of changes and I don't think we would want this to all to go in at once. If you want all of these changes you could just run your own fork.